memberships/views.py

```diff
@@ -142,14 +193,20 @@ def stripe_webhook(request):
         return HttpResponse("Failed to parse stripe payload", status=400)
 
 
-@login_required()
+# @login_required()
 def details_view(request):
+    user = request.user
+    verified = False
+    if user.member.email_verified:
+        verified = True
+
```

Comment:
Commenting out `@login_required()` means that when a user who is not logged in, `AnonymousUser`, tries to go to `/memberships/details/`, an `AttributeError` is given as there is no attribute `member` for an `AnonymousUser`.